### PR TITLE
Implement bingbong voice command

### PR DESCRIPTION
**Pull request by a-gavin**

### CODE REVIEW

File: .gitattributes
Comment (greenerino) [inline .gitattributes]: Is this added separately from the rest of the mp3 files? Why does line 2 not include this?

File: Dockerfile
Comment (greenerino) [inline Dockerfile:1]: Should this be a separate PR?

File: .gitattributes
Comment (a-gavin) [reply .gitattributes]: Good point. I'll change that ; )

File: .gitattributes
Comment (greenerino) [inline .gitattributes:1]: image

**Review by greenerino — APPROVED** [review]
Looks good! Was in a call with Alex and he showed me that it worked. Please wait for the DUCCster to review before merging

**Pull request merged**
